packages/generator/templates/app/npmrc

```diff
@@ -1,7 +1,9 @@
 save-exact=true
 legacy-peer-deps=true
-
+strict-peer-dependencies=false 
+side-effects-cache=false
 public-hoist-pattern[]=next
 public-hoist-pattern[]=secure-password
 public-hoist-pattern[]=*jest*
 public-hoist-pattern[]=@testing-library/*
+# Dont change above this
```

Comment:
hm?

Reply:
We should let the user know not to modify npmrc file above the comment. Thoughts?